marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx

```diff
@@ -380,11 +390,59 @@ const AddEditRelationship: React.FC<Props> = (props) => {
     </Select>
   );
 
+  const deleteEntityProperty= async () => {
+    let entityName = props.relationshipInfo.sourceNodeName;
+    let propertyName = props.relationshipInfo.relationshipName;
+    const entities = await primaryEntityTypes();
```

Comment:
Can we use **props.entityTypes** here like `const entities = props.entityTypes`, instead of another api call to /models/primaryEntityTypes ?